app/views/orders/new.html.erb

```diff
@@ -22,7 +22,7 @@
 <h3 align = "right">Total: <%= number_to_currency(cart.total)%></h3>
 <center>
   <h2>Shipping Info:</h2>
-  <%= form_tag "/orders", method: :create do %>
+  <%= form_tag "/profile/orders", method: :create do %>
```

Comment:
Not urgent, but creating a partial for this using `form_with` would be nice for refactoring.